site/src/modules/provisioners/ProvisionerAlert.tsx

```diff
@@ -4,30 +4,38 @@ import { AlertDetail } from "components/Alert/Alert";
 import { Stack } from "components/Stack/Stack";
 import { ProvisionerTag } from "modules/provisioners/ProvisionerTag";
 import type { FC } from "react";
+
+export enum AlertVariant {
+	Standalone = "Standalone",
+	InLogs = "InLogs",
+}
```

Comment:
You may want to comment on the differences between these two options and describe how they are rendered.